nano/node/election.cpp

```diff
@@ -3,20 +3,23 @@
 
 #include <boost/format.hpp>
 
+using namespace std::chrono;
+
 nano::election_vote_result::election_vote_result (bool replay_a, bool processed_a)
 {
 	replay = replay_a;
 	processed = processed_a;
 }
 
 nano::election::election (nano::node & node_a, std::shared_ptr<nano::block> block_a, bool const skip_delay_a, std::function<void(std::shared_ptr<nano::block>)> const & confirmation_action_a) :
```

Comment:
`skip_delay_a` was used to immediately start requesting for elections started from `active_transactions::search_frontiers`. Seems like a similar behavior can be achieved by transitioning to `active` immediately in `active_transactions::add` for these cases, currently it will take 5 seconds to start requesting.